docs/content/releases/5.2.0.md

```diff
@@ -13,32 +13,9 @@ Read more about how you can [get started]({{< relref "quickstart/_index.md" >}})
 
 ## Major Features
 
-We excited to announce v0.1 of our brand new `pgo` command line interface (CLI)!  This new versions is compatible with all currently supported PGO v5 release lines.
-
-With a focus on day two operations and disaster recovery (DR), of the `pgo` v0.1 includes the following commands:
-
-```bash
-# Create a PostgresCluster
-pgo create postgrescluster hippo
-
-# Delete a PostgresCluster
-pgo delete postgrescluster hippo
-
-# Backup a PostgresCluster
-pgo backup hippo --repoName="repo1"
-
-# View PostgresCluster Backup Information
-pgo show backup hippo --repoName="repo1"
-
-# Restore a PostgresCluster
-pgo restore hippo --repoName="repo1"
-pgo restore hippo --disable
-
-# Create a Support Export
-pgo support export hippo
-```
-
-Please note that `pgo` can either be run on it's own, or as a `kubectl` plugin.  For additional details, please see the [PGO CLi documentation]().
+This and all PGO v5 releases are compatible with a brand new `pgo` command line interface.
```

Comment:
:question: I'm wondering about this wording. For example, `pgo version` requires the CRD labels to be in place. It won't break, but the PGO version won't be shown in certain versions when certain install methods were used, iirc.

If we just mean the CLI won't blow up, then I agree, but figured it was worth mentioning.

Reply:
Looks like maybe 5.0.0–5.0.3 don't have it. https://github.com/CrunchyData/postgres-operator-examples/commit/4f123b220130cdbba210034a862e8225ce7bd8f5